### PR TITLE
Add about-texts

### PR DESCRIPTION
**Pull request by sornas**

### CODE REVIEW

**Review by th3tard1sparadox — APPROVED** [review]

**Review by vollells — APPROVED** [review]

File: website/pages/contact_se.md [hunk 1]
```diff
@@ -13,38 +13,235 @@ Om du inte redan är med i Slack-gruppen så kan du
...
+
+Mitt namn är Agnes och under året har jag suttit som LiTHe kods
+verksamhetsansvarig! 
+
+</p>
+<p class="profile-card-desc">
+
+Som verksamhetsansvarig är man kontaktperson för företag, vilket innebär lite
+planering och ganska mycket mailande. Posten är perfekt för dig som är duktig på
+att organisera och är sugen på att slipa dsina mail-skills.
```
Comment (FredTheDino) [inline website/pages/contact_se.md:197]:
```suggestion
att organisera och är sugen på att slipa dina mail-skills.
```

**Review by FredTheDino — APPROVED** [review]
LGTM, stavfel i Agnes text men annars så!

**Review by GabrielTofvesson — APPROVED** [review]
👍

File: website/pages/contact_en.md [hunk 2]
```diff
@@ -13,37 +25,235 @@ If you haven't already joined our Slack group, you can do so using the following
...
+
+Mitt namn är Agnes och under året har jag suttit som LiTHe kods
+verksamhetsansvarig! 
+
+</p>
+<p class="profile-card-desc">
+
+Som verksamhetsansvarig är man kontaktperson för företag, vilket innebär lite
+planering och ganska mycket mailande. Posten är perfekt för dig som är duktig på
+att organisera och är sugen på att slipa dsina mail-skills.
```
Comment (FredTheDino) [inline website/pages/contact_en.md:198]:
```suggestion
att organisera och är sugen på att slipa dina mail-skills.
```

**Review by FredTheDino — APPROVED** [review]

**Review by agneswendt — APPROVED** [review]

**Review by EMattfolk — APPROVED** [review]

**Pull request merged**